projects/packages/videopress/src/client/components/video-frame-selector/index.jsx

```diff
@@ -36,12 +36,17 @@ export const VideoPlayer = ( { src, setMaxDuration = null, currentTime } ) => {
 	};
 
 	return (
-		<video
-			ref={ videoPlayer }
-			muted
-			className={ styles.video }
-			onDurationChange={ onDurationChange }
-		/>
+		<div className={ styles[ 'video-player-wrapper' ] }>
+			<div className={ styles[ 'video-player-spinner-wrapper' ] }>
+				<Spinner className={ styles.spinner } />
```

Comment:
You could use [`loadeddata`](https://html.spec.whatwg.org/multipage/media.html#event-media-loadeddata) event o avoid rendering the spinner all time

Reply:
https://github.com/Automattic/jetpack/pull/26689